pkg/steps/pod.go

```diff
@@ -191,6 +192,18 @@ func PodStep(name string, config PodStepConfiguration, resources api.ResourceCon
 	}
 }
 
+func getEnv(s *podStep) []coreapi.EnvVar {
```

Comment:
can you make a PR upstream to expose the `kubeEnv` function? We will want to re-use it, and need the sorting behavior for stable test output

Reply:
Sure ... let me do the upstream PR first ...
Testing is still unstable ... :(